git-hooks/install.bash

```diff
@@ -1,5 +1,8 @@
+#! /usr/bin/env bash
+script_dir=$(dirname "${BASH_SOURCE[0]}")
+
 # install the global hooks
-git config --global core.hooksPath $(dirname "${BASH_SOURCE[0]}")/git_hooks/
+git config --global core.hooksPath $script_dir/git_hooks/
```

Comment:
You are inconsistent with creating a separate variable. If you want to, you could use $TUE_INSTALL_CURRENT_TARGET_DIR instead of creating it yourself each target.

Otherwise be consistent.

Reply:
The only places where I don't create a separate variable are the `setup` files where using `$TUE_INSTALL_CURRENT_TARGET_DIR` would not work as `tue-get install` isn't responsible for sourcing them AFAIK.